src/server/models/match.model.js

```diff
@@ -0,0 +1,28 @@
+import mongoose from 'mongoose';
+const Schema = mongoose.Schema;
+
+let matchSchema = new Schema({
+    mentorId: Schema.Types.ObjectId,
+    menteeId: Schema.Types.ObjectId,
+    requestDate: {
+        type: Date,
+        default: Date.now(),
+    },
+    isAccepted: {
+        type: Boolean,
+        default: false,
```

Comment:
If mentor accepted 'acceptDate', 'isAccepted' will be true and 'isCanceled' will be false. So 'cancelDate' wil be null and don't creat data in mongoDB.
Finally, 'acceptDate' and 'cancelDate' do not default.